packages/auth-middleware/src/middleware.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+
+import { RSAA } from 'redux-api-middleware';
+
+import type { Dispatch, Store } from 'redux';
+import type { Action, GetState } from './types';
+
+export default function authMiddleware(store: Store<GetState, Action, Dispatch<Action>>) {
+  return (next: Dispatch<Action>) => (action: Action) => {
+    const apiCall = action[RSAA];
+
+    if (!apiCall) {
+      return next(action);
+    }
+
+    const {auth} = store.getState();
+    const token = auth && auth.token ? auth.token : null;
```

Comment:
There is no need to check for `auth` it will be declared if the reducer was added properly. Also you can consider using https://babeljs.io/docs/en/babel-plugin-proposal-nullish-coalescing-operator

Reply:
Suppose this is even better: https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining